.github/workflows/deploy_tagged.yaml

```diff
@@ -1,54 +1,27 @@
 name: Reusable tagged release deploy
 
 on:
-  workflow_dispatch:
```

Comment:
I don't think this section is needed. The calling workflow should have the `workflow_dispatch` defined so this isn't actually used.